File: burbuja.cs
using System;

class burbuja
{
    static void BubbleSort(int[] a)
    {
        int s = a.Length;

        for (int i = 0; i < s; i++)
        {
            bool isSwapped = false;

            for (int j = 0; j < s - i - 1; j++)
            {
                if (a[j] > a[j + 1])
                {
                    int temp = a[j];
                    a[j] = a[j + 1];
                    a[j + 1] = temp;

                    isSwapped = true;
                }
            }

            if (!isSwapped)
                break;
        }
    }

    static void Main()
    {
        int[] a = { 15, 16, 11, 13, 14 };

        Console.WriteLine("Antes de ordenar:");
        foreach (int x in a)
            Console.Write(x + " ");

        BubbleSort(a);

        Console.WriteLine("\nDespués de ordenar:");
        foreach (int x in a)
            Console.Write(x + " ");
            
    }
}

File: burbuja.py
def bubbleSort(a):
    s = len(a)

    for i in range(s):
        isSwapped = False

        for j in range(0, s - i - 1):
            if a[j] > a[j + 1]:
                a[j], a[j + 1] = a[j + 1], a[j]
                isSwapped = True

        if isSwapped == False:
            break


a = [15, 16, 11, 13, 14]

print("Antes de ordenar:")
for x in a:
    print(x, end=" ")

bubbleSort(a)

print("\nDespués de ordenar:")
for x in a:
    print(x, end=" ")